addon/templates/components/editable-field.hbs

```diff
@@ -1,14 +1,26 @@
 <span class="content">
   {{#unless isEditing}}
-    <span onclick={{perform edit}} class="clickable editable">
+    <span class="clickable">
       {{#if value}}
         {{#if looksEmpty}}
-          {{fa-icon "edit" class="enabled"}}
+          {{fa-icon "edit"
+            class="editable enabled"
```

Comment:
Is `editable` needed here now that `[data-test-edit]` is used for tests?

Should now be `clickable`.